web/src/components/landingpage/infosLandingPage.js

```diff
@@ -0,0 +1,122 @@
+            aqueles não irão beber, teremos refrigerente e água 😔
+          </p>
+        </div>
+        <div></div>
+      </div>
+      <div className="landing-container bg-white text-secondary">
+        <div className="text-start px-auto py-5">
+          <h3 className="landing-title">Comida</h3>
+          <p className="fs-4 py-3">
+            <span className="fw-bold text-uppercase">Não</span> iremos ter comida no local, é
+            responsabilidade de cada um já vir comido. A ceia de natal já é no dia seguinte
+          </p>
+          <p className="fs-4 py-3">
+            Está liberado levar sua própria comida, mas cuidado pro amiguinho não comer.
+          </p>
+          <p className="fs-4 py-3">
+            Também há a possibilidade de pedir comida para lá, vai ser tranquilo pro pessoa dos
+            aplicativos
+          </p>
+        </div>
+        <div>
+          <img className="landing-image" src="../../images/comida.png"></img>
+        </div>
+      </div>
+      <div className="landing-container">
+        <img className="landing-image" src="../../images/dance.png"></img>
+        <div className="text-start px-auto py-5">
+          <h3 className="landing-title">Música</h3>
+          <p className="fs-4 py-3">
+            O foco de música dessa despedida é o FUNK, então podem ir preparando as dancinhas porque
+            vamos descer até o chão
+          </p>
+          <p className="fs-4 py-3">
+            Podem ficar tranquilos que tocaremos de tudo: Sertanejo, piseiro, eletrônica e por aí
+            vai
+          </p>
+          <p className="fs-4 py-3">
+            Contaremos com a presença de 2 dos melhores DJs que o planeta terra já presenciou
+          </p>
+        </div>
+        <div></div>
+      </div>
+      <div className="landing-container bg-white text-secondary">
+        <div className="text-start px-auto py-5">
+          <h3 className="landing-title">Ingresso</h3>
+          <p className="fs-4 py-3">
+            Não esqueça do seu ingresso e documento com foto no dia da despedida, esse será seu
+            único meio para conseguir embarcar nessa viagem
+          </p>
+          <p className="fs-4 py-3">
+            Só relembrando que não pode levar o amiguinho, terá segurança na porta para a checagem
+            dos ingressos, vamos evitar problemas!
+          </p>
+        </div>
+        <div>
+          <img className="landing-image" src="../../images/mocambo.png"></img>
+        </div>
+      </div>
+      <div className="landing-container flex-column text-center">
+        <p className="fs-4">Patrocinadores:</p>
+        <div className="d-flex justify-content-center">
+          <IbmLogo width="100px" height="100px" className="mx-3" />
+          <VtexLogo width="100px" height="100px" className="mx-3" />
+          <MicrossoftLogo width="100px" height="100px" className="mx-3" />
+          <ToptalLogo width="100px" height="100px" className="mx-3" />
+        </div>
```

Comment:
This is too small on mobile screens. Maybe you could change this to two rows and two cols on responsive design
![image](https://user-images.githubusercontent.com/31069709/207731228-32543ee2-319e-4fc7-a7d6-28b2447d59ae.png)